config.cfg

```diff
@@ -9,7 +9,8 @@ protocol = "both"
 domain = "auth.example.org"
 # zone name server
 nsname = "auth.example.org"
-# admin email address, where @ is substituted with .
+# admin email address, where "@" is substituted with dot (".")
+# any dots in email username must be escaped with "\", e.g. firstname\.lastname.example.com
```

Comment:
That needs to be a double-backslash:
```
firstname\\.lastname.example.com
```

This is because of TOML escaping.  If you put a single backslash here, then acme-dns fails to start with the following error:

```
Mar 02 10:51:38 ns-acme acme-dns[34272]: time="2022-03-02T10:51:38Z" level=error msg="Encountered
an error while trying to read configuration file:  Near line 6 (last key parsed 'general.nsadmin'):
invalid escape character '.'; only the following escape characters are allowed:
\\b, \\t, \\n, \\f, \\r, \\\", \\\\, \\uXXXX, and \\UXXXXXXXX"
```

Reply:
Fixed